terraform/gcp/k8s/values.yaml

```diff
@@ -0,0 +1,18 @@
+adminUrl: ""
```

Comment:
for my own understanding, since all of the values here are blank- what's the purpose of this file?

Reply:
It is the "base" values file, and your environments overlay onto this. It does two important things. First, it informs helm the structure of our `.Values` so it can give us more intelligent errors. Second, and more importantly, is that it enables autocomplete in the IDE, ahaha. 

It won't without this file because helm doesn't know where we keep our environment-specific variable files. If you think about it, helm is really meant for making charts that you send out to the public so that your application is reproducible. In that case, having an 'example, default' values file is really important. For us, it isn't so much, but it's the way helm works, so we have to make it.

Reply:
Gotcha, that makes sense. Thank you for the detailed explanation!